src/components/AppHelp.vue

```diff
@@ -183,6 +183,7 @@ table.notes-help td:first-child, table.notes-help th:first-child {
 
 table.notes-help tr:nth-child(even) {
 	background-color: #eeeeee;
+	color: black;
```

Comment:
Thanks a lot for your pull request. I think the better approach would be to adjust the background color to make sure it uses the css variable we have in place at Nextcloud which adapts to the dark/light mode dynamically.

```suggestion
	background-color: var(--color-background-dark);
```

Reply:
oh, that makes sense! I tried it and it works fine on my machine (images attached). I updated the PR.

Huge thanks!

<img width="707" alt="image" src="https://user-images.githubusercontent.com/11048462/227384346-3311a076-0f60-43d5-bdcf-f8ee9d617e2b.png">
<img width="710" alt="image" src="https://user-images.githubusercontent.com/11048462/227384351-bb241dae-6755-40fb-a428-35da1d1acc40.png">